inlong-dashboard/src/metas/groups/common/status.tsx

```diff
@@ -40,6 +40,16 @@ export const statusList: StatusProp[] = [
     value: 30,
     type: 'default',
   },
+  {
+    label: i18n.t('meta.Group.Status.Deleted'),
+    value: 40,
+    type: 'warning',
```

Comment:
The status icon needs to be standardized, the error status should uses error type.